src/api/v2/agents/handlers/join.ts

```
@@ -65,7 +77,7 @@ function buildInviteUrl(slug: string): string {
 export async function joinHandler(req: Request, res: Response) {
   // Force error responses for testing (non-production XMTP env only) — see JSDoc above for usage
   const forceError =
-    XMTP_ENV !== "production" ? req.headers["x-force-error"] : undefined;
+    shouldAllowForcedErrors() ? req.headers["x-force-error"] : undefined;
```

ESLint

[error] Replace `⏎····shouldAllowForcedErrors()·?·req.headers["x-force-error"]` with `·shouldAllowForcedErrors()⏎····?·req.headers["x-force-error"]⏎···`

(prettier/prettier)